app/bundles/CoreBundle/Config/services.php

```diff
@@ -44,6 +45,11 @@
     $services->set('mautic.http.client', GuzzleHttp\Client::class)->autowire();
     $services->set(Mautic\CoreBundle\Doctrine\MigrationFactoryDecorator::class)->autowire();
 
+    if (class_exists(StringExtension::class)) {
+        $services->set(StringExtension::class)
+            ->tag('twig.extension');
+    }
+
```

Comment:
```suggestion
    $services->set(StringExtension::class)
        ->tag('twig.extension');
```
Since `twig/string-extra` is required and used across bundles, this shouldn't be optional. Please remove the `class_exists()` check and register `StringExtension` unconditionally.